client/capstone-app/src/components/MapWithRoute.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { MapContainer, TileLayer, GeoJSON, Marker } from 'react-leaflet';
+import L from 'leaflet';
+import 'leaflet/dist/leaflet.css';
+
+const MapWithRoute = ({ routeData, currentRadius, destinationRadius, centerCoordinates}) => {
+  const onEachFeature = (feature, layer) => {
+    if (feature.properties && feature.properties.summary) {
+      layer.bindPopup(`Distance: ${feature.properties.summary.distance} meters<br>Duration: ${feature.properties.summary.duration} seconds`);
+    }
+  };
+
+  return (
+    <MapContainer center={[0,0]} zoom={15} style={{ height: '80vh', width: '100vh' }}>
+      <TileLayer
+        url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"
+        attribution='&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors'
+      />
+      <Marker position={[-90.086568, 29.956042]} />
```

Comment:
What are these coords? Imo it'd make sense to ask for your current location and then by default like point to Lagos capital if MVP is explicitly meant to target the Lagos area. 

For reference, Google maps will default to google HQ on Android if no location.

Reply:
They are supposed to be the default coords. They were hard-coded for testing purposes